### PR TITLE
i18n(es): Translated `hygraph.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `hygraph.mdx` to spanish.

### CODE REVIEW

File: src/content/docs/es/guides/cms/hygraph.mdx [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+title: Hygraph y Astro
+description: Agrega contenido a tu proyecto de Astro usando Hygraph como CMS
+type: cms
+service: Hygraph
+i18nReady: true
+---
+import FileTree from '~/components/FileTree.astro'
+
+[Hygraph](https://hygraph.com/) es un gestor de contenido federado. Expone un punto final de GraphQL para obtener contenido.
```
Comment (Waxer59) [inline src/content/docs/es/guides/cms/hygraph.mdx:10]:
```suggestion
[Hygraph](https://hygraph.com/) es un gestor de contenido federado. Expone un endpoint de GraphQL para obtener contenido.
```

File: src/content/docs/es/guides/cms/hygraph.mdx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+En esta sección, crearás un [Hygraph](https://hygraph.com/) de GraphQL para obtener contenido con Astro.
+
+### Prerrequisitos
+
+Para empezar, necesitarás lo siguiente:
+
+1. **Una cuenta y un proyecto de Hygraph**. Si no tienes una cuenta, puedes [registrarte gratis](https://app.hygraph.com/signup) y crear un nuevo proyecto.
+
+2. **Permisos de acceso a Hygraph** - En `Project Settings > API Access`, configura los permisos de la API de contenido público para permitir solicitudes de lectura sin autenticación. Si no has configurado ningún permiso, puedes hacer clic en **Sí, inicializar valores predeterminados** para agregar los permisos necesarios para usar la "API de contenido de alto rendimiento".
```
Comment (Waxer59) [inline src/content/docs/es/guides/cms/hygraph.mdx:22]:
```suggestion
2. **Permisos de acceso a Hygraph** - En `Project Settings > API Access`, configura los permisos de la API de contenido público para permitir solicitudes de lectura sin autenticación. Si no has configurado ningún permiso, puedes hacer clic en **Yes, initialize defaults** para agregar los permisos necesarios para usar la "API de contenido de alto rendimiento".
```

File: src/content/docs/es/guides/cms/hygraph.mdx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+### Configurando el endpoint
+
+Para agregar tu endpoint de Hygraph a Astro, crea un archivo `.env` en la raíz de tu proyecto con la siguiente variable:
+
+```ini title=".env"
+HYGRAPH_ENDPOINT=YOUR_HIGH_PERFORMANCE_API
```
Comment (Waxer59) [inline src/content/docs/es/guides/cms/hygraph.mdx:29]:
```suggestion
HYGRAPH_ENDPOINT=TU_API_DE_ALTO_RENDIMIENTO
```

File: src/content/docs/es/guides/cms/hygraph.mdx [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+}
+```
+
+:::note
+Lee más sobre [el uso de variables de entorno](/es/guides/environment-variables/) y archivos `.env` en Astro.
+:::
+
+Tu directorio raíz ahora debería incluir estos nuevos archivos:
+
+<FileTree title="Project Structure">
```
Comment (Waxer59) [inline src/content/docs/es/guides/cms/hygraph.mdx:48]:
```suggestion
<FileTree title="Estructura del proyecto">
```

File: src/content/docs/es/guides/cms/hygraph.mdx [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+- **.env**
+- astro.config.mjs
+- package.json
+</FileTree>
+
+### Obteniendo datos
+ 
+Obtén los datos de tu proyecto de Hygraph usando `HYGRAPH_ENDPOINT`.
+
+Por ejemplo, para obtener las entradas de un tipo de contenido `blogPosts` que tiene un campo de cadena `title`, crea una consulta GraphQL para obtener ese contenido. Luego, define el tipo de contenido y configúralo como el tipo de respuesta.
```
Comment (Waxer59) [inline src/content/docs/es/guides/cms/hygraph.mdx:60]:
```suggestion
Por ejemplo, para obtener las entradas de un tipo de contenido `blogPosts` que tiene un campo de tipo string `title`, crea una consulta GraphQL para obtener ese contenido. Luego, define el tipo de contenido y configúralo como el tipo de respuesta.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the great work 🚀

**Pull request merged**